benchmarks/CMakeLists.txt

```diff
@@ -40,7 +40,10 @@ if(NOT WIN32)
   add_subdirectory(develop)
   add_subdirectory(union_find)
 endif()
-add_subdirectory(triangulated_surface_distance)
+
+if(Kokkos_VERSION VERSION_GREATER_EQUAL 4.3.99)
```

Comment:
I added this so that I can use `Kokkos::Array<int, 3>` for triangles and copy to device without an issue.